.github/workflows/build.yml

```diff
@@ -26,16 +26,16 @@ jobs:
           - TealiumAutotracking
     steps:
       - name: Setup Xcode version
-        uses: maxim-lobanov/setup-xcode@v1.3.0
+        uses: maxim-lobanov/setup-xcode@v1.4.1
         with:
-          xcode-version: latest-stable
+          xcode-version: '13.3'
```

Comment:
is this required?
I feel like `latest-stable` should work and should also be the preferred value to have here

Reply:
Im not sure about xcode-version, but for some reason using latest-stable macos version would not pull in xcode 13 so I had to manually specify a macos version, so that will need to be changed later also so I don't think it would hurt to leave this

Reply:
in the past i ended up having the same problems you described an d wrongly changed the xcode-version, that's why i'm asking. 
If you are not sure about latest-stable i would try if that works, otherwise i would revert and just use 13.3.1 in everyone instead. 
But i'm pretty confident it should work once you updated to macos-12